ext/drbbdup/drbbdup.h

```diff
@@ -257,8 +258,32 @@ typedef struct {
      * thread before it becomes a candidate for dynamic generation.
      */
     ushort hit_threshold;
+    /**
+     * Determines whether drbbdup should track a variety of statistics. Note keeping track
+     * of statistics incurs additional overhead and it is not recommended at deployment.
+     *
+     * Set to true if the client calls drbbdup_get_stats().
+     */
+    bool is_stat_enabled;
 } drbbdup_options_t;
 
+/**
+ * Various statistics related to drbbdup.
+ */
+typedef struct {
+    /** Number of fragments which have case handling turned off. */
```

Comment:
Have a struct size parameter, like all the similar interface structs do (dr_stats_t, drwrap_stats_t), so we can safely append to this in the future.

Reply:
Done.